packages/integration-tests/suites/replay/errors/droppedError/test.ts

```diff
@@ -0,0 +1,124 @@
+import { expect } from '@playwright/test';
+
+import { sentryTest } from '../../../../utils/fixtures';
+import {
+  expectedClickBreadcrumb,
+  expectedConsoleBreadcrumb,
+  getExpectedReplayEvent,
+} from '../../../../utils/replayEventTemplates';
+import {
+  getReplayEvent,
+  getReplayRecordingContent,
+  shouldSkipReplayTest,
+  waitForReplayRequest,
+} from '../../../../utils/replayHelpers';
+
+sentryTest(
+  '[error-mode] should start recording if an error occurred although the error was dropped',
+  async ({ getLocalTestPath, page }) => {
+    if (shouldSkipReplayTest()) {
+      sentryTest.skip();
+    }
+
+    let callsToSentry = 0;
+    const reqPromise0 = waitForReplayRequest(page, 0);
+    const reqPromise1 = waitForReplayRequest(page, 1);
+    const reqPromise2 = waitForReplayRequest(page, 2);
+
+    await page.route('https://dsn.ingest.sentry.io/**/*', route => {
+      callsToSentry++;
+      return route.fulfill({
+        status: 200,
+        contentType: 'application/json',
+        body: JSON.stringify({ id: 'test-id' }),
+      });
+    });
+
+    const url = await getLocalTestPath({ testDir: __dirname });
+
+    await page.goto(url);
+    await page.click('#go-background');
+    expect(callsToSentry).toEqual(0);
+
+    await page.click('#error');
+    const req0 = await reqPromise0;
+
+    await page.click('#go-background');
+    const req1 = await reqPromise1;
+    expect(callsToSentry).toEqual(2); // 2 replay events
+
+    await page.click('#log');
+    await page.click('#go-background');
+    const req2 = await reqPromise2;
+
+    const event0 = getReplayEvent(req0);
+    const content0 = getReplayRecordingContent(req0);
+
+    const event1 = getReplayEvent(req1);
+    const content1 = getReplayRecordingContent(req1);
+
+    const event2 = getReplayEvent(req2);
+    const content2 = getReplayRecordingContent(req2);
+
+    expect(event0).toEqual(
+      getExpectedReplayEvent({
+        contexts: { replay: { error_sample_rate: 1, session_sample_rate: 0 } },
+        // This is by design. A dropped error shouldn't be in this list.
+        error_ids: [],
+        replay_type: 'error',
+      }),
+    );
+
```

Comment:
Should we stop the test here, I wonder? As I guess everything below here is kind of a duplicate of the other test(s) about error --> session mode switch?